test/gui/tst_syncing/test.feature

```diff
@@ -401,7 +425,15 @@ Feature: Syncing files
         And the user sets the temp folder "~`!@#$^&()-_=+{[}];',)PRN%" as local sync path in sync connection wizard
         And the user selects "ownCloud" as a remote destination folder
         And the user selects the following folders to sync:
-            | folder                 |
-            | ~`!@#$^&()-_=+{[}];',) |
+            | folder                  |
+            | ~`!@#$^&()-_=+{[}];',)  |
+            | simple-folder           |
+            | test-folder/sub-folder2 |
         Then the folder "~`!@#$^&()-_=+{[}];',)" should exist on the file system
+        And the folder "simple-folder" should exist on the file system
         But the folder "~test%" should not exist on the file system
+        When user "Alice" deletes the folder "simple-folder" in the server
+        And the user waits for the files to sync
+        Then the folder "simple-folder" should not exist on the file system
```

Comment:
it has been covered in above scenario

Reply:
In above scenario, it is for deleting folder from file system and this step is for deleting folder from the server.

Reply:
ahh! :+1: